examples/sampling/pyg/node_classification.py

```diff
@@ -144,35 +167,37 @@ def train(model, dataloader, optimizer, criterion, device, num_classes):
     num_batches = 0  # Counter for the number of mini-batches processed
 
     for minibatch in dataloader:
-        node_features = minibatch.node_features["feat"]
-        labels = minibatch.labels
+        pyg_data = minibatch.to_pyg_data()
```

Comment:
add a few comments here about converting GB.MiniBatch to pyg ones. Just highlight the key calls.